### PR TITLE
Sub actual build dir for literal '${build_dir}'

### PR DESCRIPTION
**Pull request by freeformz**
With the removal of the /app/code symlink, this allows a user to specify
paths that are relative to the build (`./` appears to not work).

I am not sure this is the _best_ way to support this, but seems better than relying on "/app"

### CODE REVIEW

**Comment by freeformz** [comment]
I need to improve the CGO test before merging anyway.

**Comment by amerine** [comment]
Seems fine to me.

**Comment by freeformz** [comment]
Also, tests added.

File: README.md [hunk 1]
```diff
@@ -111,11 +111,13 @@ $ make test
...
-This buildpack supports building with C dependencies via [cgo][cgo]. You can set
-config vars to specify CGO flags to specify paths for vendored dependencies. For
-example, if you added C headers to and `includes` directory, add the config var
-`CGO_CFLAGS` with the value `-I/app/code/includes` and include the relevant
-header files in `includes/` in your app.
+The buildpack supports building with C dependencies via [cgo][cgo]. You can set
+config vars to specify CGO flags to specify paths for vendored dependencies. The
+literal text of `${build_dir}` will be replaced with the directory the build is
+happening in. For example, if you added C headers to an `includes/` directory,
+add the following config to your app: `heroku config:set CGO_CFLAGS'-I${
```
Comment (msakrejda) [inline README.md]: There should be a space here after `CGO_FLAGS`, no?

Comment (freeformz) [inline README.md]: should be an `=`. Fixed. Thanks

Comment (msakrejda) [inline README.md]: Oh right you are. For some reason thought it was two separate args.

**Pull request merged**